test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1082,13 +1082,46 @@ void UcxConnection::set_log_prefix(const struct sockaddr* saddr,
     ss << "[UCX-connection " << this << ": #" << _conn_id << " "
        << _remote_address << "]";
     memset(_log_prefix, 0, MAX_LOG_PREFIX_SIZE);
+    memset(_conn_log_prefix, 0, MAX_LOG_PREFIX_SIZE);
     int length = ss.str().length();
     if (length >= MAX_LOG_PREFIX_SIZE) {
         length = MAX_LOG_PREFIX_SIZE - 1;
     }
     memcpy(_log_prefix, ss.str().c_str(), length);
 }
 
+void UcxConnection::set_conn_log_prefix()
+{
+    ucp_ep_attr_t ep_attr;
+    std::string local_address;
+    std::string remote_address;
+    std::stringstream ss;
```

Comment:
using `std::stringstream` looks redundant here, I would suggent to use string operators `+` and `+=`

Reply:
done